assets/css/style.css

```diff
@@ -213,3 +228,164 @@ body {
   background: #9490ff;
   color: #fff;
 }
+
+/* Works section */
+.work {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 120px 0;
+}
+
+.worktitle h1 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  text-align: left;
+  position: relative;
+  left: 0;
+  color: #7d73fc;
+  flex: none;
+  flex-grow: 0;
+  margin: 24px 0;
+  width: 327px;
+}
+
+.work .divider {
+  border: 1px solid #646369;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+}
+
+.card {
+  width: 327px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+
+.card .profile {
+  position: static;
+  width: 327px;
+  height: 236px;
+  left: 0;
+  top: 0;
+  background: linear-gradient(0deg, #f9f9f9, #f9f9f9), url(image_processing20200716-21735-f07s4k.png);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.card h3 {
+  width: 327px;
+  height: 103px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.card .workText {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 327px;
+}
+
+.card #firstTitle {
+  background: url(../images/EllipseFT.svg);
+  background-size: 150px 482px;
+  background-repeat: no-repeat;
+  background-position-y: 100%;
+  background-position-x: 0%;
+}
```

Comment:
- [ ] Kindly make sure that the background touches the edge of the page as shown in the design as shown in the Figma design
- Figma
<img width="399" alt="Screen Shot 2022-03-18 at 16 38 22" src="https://user-images.githubusercontent.com/28307960/159023479-ba25115f-4067-40eb-8da9-0ff5d9edb0b5.png">
- Your design
<img width="351" alt="Screen Shot 2022-03-18 at 16 34 35" src="https://user-images.githubusercontent.com/28307960/159023538-3138a070-5a45-4da7-9af6-8779c26abd23.png">

Reply:
Fixed background